fe/fe-core/src/main/java/org/apache/doris/nereids/trees/expressions/functions/executable/DateTimeArithmetic.java

```diff
@@ -210,14 +210,6 @@ public static Expression secondsAdd(DateTimeV2Literal date, IntegerLiteral secon
         return date.plusSeconds(second.getValue());
     }
 
-    /**
-     * datetime arithmetic function microseconds-add.
-     */
-    @ExecFunction(name = "microseconds_add", argTypes = {"DATETIMEV2", "INT"}, returnType = "DATETIMEV2")
```

Comment:
restore this and make `DateTimeV2Literal.plusMicroseconds` always use MICRO pricision